suite-native/module-send/src/screens/SendReviewScreen.tsx

```diff
@@ -0,0 +1,115 @@
+import { useDispatch, useSelector } from 'react-redux';
+
+import { CommonActions } from '@react-navigation/native';
+import { isRejected } from '@reduxjs/toolkit';
+
+import {
+    AppTabsRoutes,
+    GoBackIcon,
+    RootStackRoutes,
+    Screen,
+    ScreenSubHeader,
+    SendStackParamList,
+    SendStackRoutes,
+    StackProps,
+} from '@suite-native/navigation';
+import { Button, Text, VStack } from '@suite-native/atoms';
+import {
+    AccountsRootState,
+    cancelSignSendFormTransactionThunk,
+    selectAccountByKey,
+    selectSendSignedTx,
+} from '@suite-common/wallet-core';
+import { useToast } from '@suite-native/toasts';
+import { AccountKey } from '@suite-common/wallet-types';
+
+import { sendTransactionAndCleanupSendFormThunk } from '../sendFormThunks';
+import { ReviewOutputItemList } from '../components/ReviewOutputItemList';
+
+const navigateToAccountDetail = ({ accountKey }: { accountKey: AccountKey }) =>
+    // Reset navigation stack to the account detail screen with HomeStack as a previous step, so the user can navigate back there.
+    CommonActions.reset({
+        index: 1,
+        routes: [
+            {
+                name: RootStackRoutes.AppTabs,
+                params: {
+                    screen: AppTabsRoutes.HomeStack,
+                },
+            },
+            {
+                name: RootStackRoutes.AccountDetail,
+                params: {
+                    accountKey,
+                },
+            },
+        ],
+    });
+
+export const SendReviewScreen = ({
+    route,
+    navigation,
+}: StackProps<SendStackParamList, SendStackRoutes.SendReview>) => {
+    const { accountKey } = route.params;
+    const dispatch = useDispatch();
+    const { showToast } = useToast();
+
+    const signedTransaction = useSelector(selectSendSignedTx);
+    const account = useSelector((state: AccountsRootState) =>
+        selectAccountByKey(state, accountKey),
+    );
+
+    if (!account) return;
+
+    const handleSendTransaction = async () => {
+        // TODO: error handling
+        if (!signedTransaction) return;
```

Comment:
I believe this should be in outer scope (similar to account condition) to return null (or better yet - ErrorMessage from atoms) if no transaction was found by selector? This function should be called just when there is a transaction to be sent IMO and we just handle reject from sending it.

Reply:
Most of the lifecycle of this component, there should be no `signedTransaction`. The component first show the review steps and only if user confirms all of them, the signed transaction is created.

Reply:
But the button that triggers this function already checks that `signedTransaction` is present. So I think you can just remove this.

Reply:
Yes, it does, but Typescript does not know this. I have to assert that the value is present to make the linter happy.

Reply:
Of course, I didn't realise. In that case, I'd consider putting the text and button in a new component. It will be better to have it there because: 
- you can send signedTransaction as prop that is required
- it would accept accountKey, so you will have access to account

I believe that is better and you can keep this function without redundant condition, that is already checked in JSX.

Reply:
Fixed in [affdc58](https://github.com/trezor/trezor-suite/pull/12968/commits/affdc5880efe6a9efaddd99b05b742b2a753ebbb) by moving the button to the separate component.